client/dashboard/sites/overview/index.tsx

```diff
@@ -144,7 +144,7 @@ function SiteOverview( {
 					spacing={ spacing }
 					alignment="flex-start"
 				>
-					<LatestActivitiesCard />
+					<LatestActivityCard site={ site } compact={ isSmallViewport } />
```

Comment:
`DomainsCard` uses `isCompact`. I kinda like `compact` better, but we should use the same prop name I think.

```suggestion
					<LatestActivityCard site={ site } isCompact={ isSmallViewport } />
```

Reply:
Makes sense; fixed in 68c6605f6831211d3aa9f5ba6fb00c8b9e93e1ff